packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/IPTransfer.tsx

```diff
@@ -435,6 +444,16 @@ const LinodeNetworkingIPTransferPanel: React.FC<CombinedProps> = (props) => {
     setError(undefined);
     setSuccessMessage('');
 
+    const noActionSelected = !Object.values(ips).find(
+      (ip) => ip.mode !== 'none'
+    );
+    if (noActionSelected) {
+      setError('Please select an action.');
```

Comment:
What do you think about keeping `error` in the `APIError` shape instead of allowing a string?

```suggestion
      setError([{ reason: 'Please select an action.' }]);
```

Reply:
Sure thing - it's a little odd but cleaner not to handle strings in the rendering